i.sentinel_2.ndvidiff/i.sentinel_2.ndvidiff.html

```diff
@@ -0,0 +1,71 @@
+<h2>DESCRIPTION</h2>
+
+<em>i.sentinel_2.ndvidiff</em> is a GRASS addon that calculates an NDVI difference map from Sentinel-2 data.
+
+<p>
+The goal of the addon is to compare the NDVI status from two time intervals for an AOI given by the vector map <em>aoi</em>. For each interval, <br>
+Sentinel-2 data is imported from the local file system (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.import</a>), cloud-masked (using <a href="https://github.com/mundialis/t.sentinel">t.sentinel.mask</a>),<br>
+    and temporally aggregated to receive one spectral band (using <a href="https://github.com/mundialis/t.rast.mosaic">t.rast.mosaic</a>) per time interval.
+<p>
+One NDVI map for each time interval as well as a difference map between the two are calculated. The user may define <br>
+a threshold via the <em>ndvi_diff_threshold</em> option, which is used to extract areas of signficant NDVI loss as raster and/or vector maps. <br>
+<p>
+The outputs of the module (RGBI mosaics, NDVI maps, NDVI difference map, NDVI loss areas) can be saved as GRASS raster/vector maps or <br>
+exported using the <em>output_dir</em> option.
+
+
+<h2>NOTES</h2>
+The <em>offset</em> option can be used to take the systematic reflectance offset into account that is added to Sentinel-2 <br>
+reflectance data since Processing Baseline 4.0.0.
+<p>
+If no <em>ndvi_diff_threshold</em>  is given, a threshold is automatically calculated from the NDVI difference map using <br>
+<em>thresh = Q1 - 1.5 * (Q3 - Q1)</em>, where Q1 and Q3 are the first and third quantile of the NDVi difference map.
+<p>
+For cloud masking, the Sen2Cor cloud mask delivered with L2A data is combined with the output of <a href="https://github.com/mundialis/t.sentinel">t.sentinel.mask</a>.
+<p>
+The temporal aggregation method to be passed on to <a href="https://github.com/mundialis/t.rast.mosaic">t.rast.mosaic</a> can be defined via the <em>aggregation_method</em> option.<br>
+Best results were achieved with <em>minimum</em> aggregation which reduces negative effects due to remaining clouds.
```

Comment:
I think it depends what it is aggregated. In this case, it is reflectance values - they are typically high in clouded areas so taking the minimum minimizes the risk of still having a cloud in there. If we aggregate NDVI values, it might make sense to take the maximum as clouds typically have low NDVI values.